layouts/partials/about/member-role.html

```diff
@@ -0,0 +1,5 @@
+<ul>
+    {{ range .Params.roles }}
+    <li class="role">{{ . }}</li>
```

Comment:
Look at the CSS Guidelines for naming.

The ul should have a class called RoleList and li should have a class called RoleList-Element.

Reply:
should we even use a list for the roles? I think a subheader would work to

Reply:
I think a list is the right choice. In the end we want to format this list as the badges were. I think from a CSS standpoint its much better to be explicit.

Reply:
We must think about if we are using the semantic elements correctly, please check if we are using those correctly with this approach.